pages/common/abduco.md

```diff
@@ -0,0 +1,23 @@
+# abduco
+
+> Terminal session manager.
+
+- List sessions:
+
+`abduco`
+
+- Attach to a session, creating it if it doesn't exist:
+
+`abduco -A {{name}} bash`
```

Comment:
I think bash should be in tokens too ? Or does it not support anything other than bash ?

Reply:
I thought `abduco -A {{name}} {{command}}` would not be very clear.
I will take whatever you choose.

Reply:
You don't have to change it to `{{command}}`! `{{bash}}` would be fine. In this case, a concrete example is great, @Jipok :-)